ob_v3p0/impl/linked-data-proof.md

```diff
@@ -0,0 +1,247 @@
+## Linked Data Proof Test Vector
+
+This chapter is an example of the signing process of a given credential with an Linked Data Proof producing a `DataIntegrityProof` of a public key expressed in `eddsa-2022` format.
+
+### Key pair & Multikey
+
+For this example we are using the following keypair:
+* Public key (hex): `4bdeafde2ea8beefadd8c699b5c7e0704cf51154d52e17b20b71337ca04cc5a5`
+* Private key (hex): `6241a409e6707bb640a0140a8a32bc3d193c33a661747284d6adfa4ed4180be44bdeafde2ea8beefadd8c699b5c7e0704cf51154d52e17b20b71337ca04cc5a5`
+
+The `MultiKey` used in this example is as follows:
+```json
+ {
+  id: 'https://example.com/issuers/876543#z6MkjZRZv3aez3r18pB1RBFJR1kwUVJ5jHt92JmQwXbd5hwi',
+  controller: 'https://example.com/issuers/876543',
+  publicKey: Uint8Array(32) [
+     75, 222, 175, 222,  46, 168, 190,
+    239, 173, 216, 198, 153, 181, 199,
+    224, 112,  76, 245,  17,  84, 213,
+     46,  23, 178,  11, 113,  51, 124,
+    160,  76, 197, 165
+  ],
+  secretKey: Uint8Array(64) [
+     98,  65, 164,   9, 230, 112, 123, 182,  64, 160,  20,
+     10, 138,  50, 188,  61,  25,  60,  51, 166,  97, 116,
+    114, 132, 214, 173, 250,  78, 212,  24,  11, 228,  75,
+    222, 175, 222,  46, 168, 190, 239, 173, 216, 198, 153,
+    181, 199, 224, 112,  76, 245,  17,  84, 213,  46,  23,
+    178,  11, 113,  51, 124, 160,  76, 197, 165
+  ],
+  publicKeyMultibase: 'z6MkjZRZv3aez3r18pB1RBFJR1kwUVJ5jHt92JmQwXbd5hwi',
+  secretKeyMultibase: 'zrv2bqTbNwCTsRrHFcJCPjVAduh4Ezcnoq1A3ZxH1GWTNkxipLVuaAoMFmze2gFN9oNXfJjufxSHWVZzsJiUsMHFMcx',
+  revoked: undefined,
+  export: [AsyncFunction: export],
+  signer: [Function: signer],
+  verifier: [Function: verifier]
```

Comment:
Is this intended to be double-quoted valid JSON? Doesn't quite look like it. Maybe the annotation could be changed to `js` instead to enable proper syntax highlighting.

Reply:
Sorry @ottonomy, you're right. This code is in js, not json